docs/samples/contact-center/app.js

```diff
@@ -103,14 +106,50 @@ function initWebex(e) {
 
 credentialsFormElm.addEventListener('submit', initWebex);
 
+
 function register() {
     webex.cc.register().then((data) => {
         console.log('Event subscription successful: ', data);
-    }).catch(() => {
-        console.log('Event subscription failed');
+        const agentProfile = webex.cc.getAgentProfileData();
```

Comment:
Register should take care of getting the agent profile, we shouldn;t need to invoke this method separately, it is not supposed to be a public method. Through the event or promise resolution, agent profile should be received